packages/scene-composer/src/components/three-fiber/DataOverlayComponent/__tests__/DataOverlayComponentSnap.spec.tsx

```diff
@@ -40,6 +41,20 @@ describe('DataOverlayComponent', () => {
   const mesh = new Mesh(geometry);
   group.add(mesh);
 
+  class ResizeObserver {
+    observe = jest.fn();
+    unobserve = jest.fn();
+    disconnect = jest.fn();
+  }
+
+  const Layout: React.FC = () => {
+    return (
+      <Canvas>
```

Comment:
why do we need Canvas here?

Reply:
Introduction of `useFrame` requires being in a canvas for testing.

Reply:
This was also fixed with the introduction of mocking `useFrame`.